cmd/testscript/os.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"errors"
+	"os"
+	"runtime"
+)
+
+// os.UserHomeDir() is implemented in
+// https://go-review.googlesource.com/c/go/+/139418 so implement here for now
+func userHomeDir() (string, error) {
```

Comment:
This seems to be used only in tests, so maybe move it into a test file?

Reply:
It's now removed so don't even need to decide where to put it.